### PR TITLE
Feat/conversation webhooks

### PR DESCRIPTION
**Pull request by Dovchik**
This time tests are utilizing mocks from open-api-spec in doppleganger, and not manual, see https://github.com/sinch/doppelganger/pull/10

### CODE REVIEW

File: tests/Sinch.Tests/e2e/Conversation/WebhooksTests.cs
Comment (asein-sinch) [inline tests/Sinch.Tests/e2e/Conversation/WebhooksTests.cs:1]: The tests are failing :'(

File: src/Sinch/SMS/DeliveryReports/Get/GetDeliveryReportResponse.cs
Comment (asein-sinch) [inline src/Sinch/SMS/DeliveryReports/Get/GetDeliveryReportResponse.cs:1]: Why do you need to manage these warning for SMS in this PR? Is it link to a new configuration?

File: src/Sinch/Conversation/Webhooks/WebhookTrigger.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public static readonly WebhookTrigger ContactMerge = new("CONTACT_MERGE");
+    public static readonly WebhookTrigger ContactUpdate = new("CONTACT_UPDATE");
+    public static readonly WebhookTrigger Unsupported = new("UNSUPPORTED");
+    public static readonly WebhookTrigger OptIn = new("OPT_IN");
+    public static readonly WebhookTrigger OptOut = new("OPT_OUT");
+    public static readonly WebhookTrigger Capability = new("CAPABILITY");
+    public static readonly WebhookTrigger ChannelEvent = new("CHANNEL_EVENT");
+    public static readonly WebhookTrigger ConversationDelete = new("CONVERSATION_DELETE");
+    public static readonly WebhookTrigger ContactIdentitiesDuplication = new("CONTACT_IDENTITIES_DUPLICATION");
+    public static readonly WebhookTrigger SmartConversations = new("SMART_CONVERSATIONS");
```
Comment (asein-sinch) [inline src/Sinch/Conversation/Webhooks/WebhookTrigger.cs:34]: I'm not sure both `SMART_CONVERSATION` and `SMART_CONVERSATIONS` exist. Can you double-check?

File: tests/Sinch.Tests/e2e/Conversation/WebhooksTests.cs
Comment (Dovchik) [reply tests/Sinch.Tests/e2e/Conversation/WebhooksTests.cs:1]: That's expected, new doppleganger haven't been merged in when this PR was created

File: src/Sinch/SMS/DeliveryReports/Get/GetDeliveryReportResponse.cs
Comment (Dovchik) [reply src/Sinch/SMS/DeliveryReports/Get/GetDeliveryReportResponse.cs:1]: The warning comes from other forced push PR with generation of xml documentation file in nuget package, here I disabling the warning cause it's not relevant.

File: src/Sinch/Conversation/Webhooks/WebhookTrigger.cs (hunk 1 above)
Comment (Dovchik) [reply src/Sinch/Conversation/Webhooks/WebhookTrigger.cs:34]: That's what I get from oas spec, 
`SMART_CONVERSATION` exists but it's in beta 
image

Comment (Dovchik) [reply src/Sinch/Conversation/Webhooks/WebhookTrigger.cs:34]: I'll remove plural variant

**Review by asein-sinch — APPROVED** [review]

**Pull request merged**